### PR TITLE
Sync charts with istio/operator

### PR DESCRIPTION
**Pull request by ostromart**
Mostly adding missing separators. Only non-trivial change is the creation of the namespace.

### CODE REVIEW

File: crds/templates/crds.yaml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# To prevent accidental injection into istio-system.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
```
Comment (howardjohn) [inline crds/templates/crds.yaml]: we may want to add all the namespaces here not just .Release.Namespace, can be in the future though

Comment (ostromart) [reply crds/templates/crds.yaml]: That's a good idea - added.

Comment (costinm) [reply crds/templates/crds.yaml]: Please don't add namespaces to a file names 'crds...'

Comment (ostromart) [reply crds/templates/crds.yaml]: Done.

File: crds/templates/namespaces.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# To prevent accidental injection into istio control plane namespaces.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
+  labels:
+    istio-operator-managed: Reconcile
```
Comment (costinm) [inline crds/templates/namespaces.yaml]: Unrelated  to this PR: but the pattern should be to specify which operator is managing this namespace ( if that's what we want ). 

"Reconcile" is quite confusing.

Comment (ostromart) [reply crds/templates/namespaces.yaml]: I've created https://github.com/istio/istio/issues/15825 to track this.

**Pull request merged**